src/ontology/mondo-edit.obo

```diff
@@ -298874,7 +298869,6 @@ xref: UMLS:C3280428 {source="NCIT:C119677", source="MONDO:equivalentTo", source=
 is_a: MONDO:0019233 {source="https://github.com/monarch-initiative/mondo/issues/1196", source="https://www.clinicalgenome.org/affiliation/40049/"} ! disorder of peroxisomal beta oxidation
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/451 {source="MONDO:mim2gene_medgen"} ! AMACR
 property_value: confidence "5.666666666666666" xsd:double
-property_value: excluded_subClassOf "'congenital bile acid synthesis defect 4'" xsd:string {source="Orphanet:79095/btnt"}
```

Comment:
I'm not sure why this was present because I don't see it on master, so I deleted it